iceoryx_examples/iceoptions/README.md

```diff
@@ -0,0 +1,68 @@
+# iceoptions
+
+## Introduction
+
+This example demonstrates what kind of quality of service options can be configured on the publisher and subscriber
+side. The options can be used for both, typed and untyped API flavours.
+
+## Expected output
+
+<!-- Add asciinema link here -->
+
+## Code walkthrough
+
+### Publisher
+
+In order to configure a publisher, we have to supply a struct of the type `iox::popo::PublisherOptions` as a second parameter.
+
+`historyCapacity` will enable subscribers to read the last n topics e.g. in case they are started later than the publisher:
```

Comment:
last n samples.

Could you please reread the whole documentation and look for the word topic and replace it with sample when it makes sense.